COVERAGE.md

```
@@ -740,302 +295,160 @@
 ```
 
 % Coverage report from v8
--------------------|---------|----------|---------|---------|-------------------
-File               | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
--------------------|---------|----------|---------|---------|-------------------
-All files          |   77.24 |        0 |       0 |   77.24 |                   
- ...dPluginTevm.js |     100 |      100 |     100 |     100 |                   
- index.js          |       0 |        0 |       0 |       0 | 1-34              
--------------------|---------|----------|---------|---------|-------------------
+----------|---------|----------|---------|---------|-------------------
+File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)